site/index.html

```diff
@@ -16,7 +16,7 @@
             </div>
             <div class="output_container">
                 <div class="frame">
-                    <label for="frame">Style: </label>
+                    <label for="output_css_selection">Style: </label>
```

Comment:
Should this be outputCssSelection?

Reply:
So far we are using snake_case for ids and classes. I would not mix the naming style.

Reply:
Okay i thought this was a javascript name 👍